packages/cli/src/commands/repo/build.ts

```diff
@@ -120,13 +125,11 @@ export async function command(cmd: Command): Promise<void> {
   await buildPackages(options);
 
   if (cmd.all) {
-    const apps = bundledPackages.filter(
-      pkg => pkg.packageJson.backstage?.role === 'app',
-    );
-
     console.log('Building apps');
-    await Promise.all(
-      apps.map(async pkg => {
+    await runParallelWorkers({
+      items: apps,
+      parallelismFactor: 1 / 2,
```

Comment:
why is the factor 1/2 if you don't run the app and backend builds concurrently? 🤔

Reply:
Would result in running up to 4 webpack builds in parallel by default, which I thought was a bit too much x)

In the end pretty much every project is most likely only building a single thing here though. We could potentially kick off the webpack build in parallel with the package builds though, but think I'll leave that as a future optimization as doing it properly is a bit more involved.